sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Listeners/BlobListenerFactory.cs

```diff
@@ -164,6 +171,7 @@ public async Task<IListener> CreateAsync(CancellationToken cancellationToken)
             };
 
             sharedBlobQueueListener.Register(_functionDescriptor.Id, registration);
+            sharedBlobQueueListener.Register(_functionDescriptor.ShortName, registration);
```

Comment:
Why is this required?

Reply:
this is to simplifying manual creation of the eventgrid callback uri.

Reply:
Is this worth making this runtime change just to simplify the testing?

Reply:
It's for consistency with EventGrid extension:
https://{functionappname}.azurewebsites.net/runtime/webhooks/_eventgrid_?functionName={functionname}&code={systemkey} (https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-event-grid-trigger?tabs=csharp%2Cbash#version-2x-and-higher-runtime)
where `{functionname} `is short function name (without namespace)
So for blobs we want the same format
https://{functionappname}.azurewebsites.net/runtime/webhooks/_blobs_?functionName={functionname}&code={systemkey}

So we need the registration to find the function to execute by short name